capa/helpers.py

```diff
@@ -85,6 +89,40 @@ def get_format(sample: str) -> str:
     return FORMAT_UNKNOWN
 
 
+@contextlib.contextmanager
+def redirecting_print_to_tqdm(disable_progress):
```

Comment:
can you also use this function from the linter script?

Reply:
no, remove the function in the linter (https://github.com/mandiant/capa/blob/7ab083f19a77f7d6ab658a0f614e6154476aca17/scripts/lint.py#L870) and use the copy in helpers.py instead.

Reply:
Will make some changes